application/src/main/java/org/togetherjava/tjbot/commands/tags/TagManageCommand.java

```diff
@@ -156,6 +156,7 @@ private void rawTag(@NotNull SlashCommandEvent event) {
         event
             .replyEmbeds(new EmbedBuilder()
                 .setDescription(MessageUtils.escapeMarkdown(tagSystem.getTag(id).orElseThrow()))
+                .setFooter(event.getUser().toString())
```

Comment:
Dont toString the user, you wont get what you expect. Do what the code did in the past, namely getting the name of the user.